src/soft-tabs-transform.coffee

```diff
@@ -0,0 +1,26 @@
+Point = require './point'
+Region = require './region'
+_ = require 'underscore-plus'
+
+module.exports =
+class SoftTabsTransform
+  constructor: (@tabLength) ->
+    @tab = _.multiplyString(" ", @tabLength)
```

Comment:
What do you think about the name `softTabString` or something a bit more verbose?